.circleci/config.yml

```diff
@@ -245,25 +247,34 @@ jobs:
             # update 301 redirect for the /latest/* route.
             aws s3api put-bucket-website --cli-input-json file://.circleci/s3-website-config.json
 
-  deploy-s3:
+  deploy-dev:
+    # To reduce chance of deployment issues, try to keep the steps here as
+    # similar as possible to the steps in deploy-release!
     resource_class: small
 
     docker:
-      - image: circleci/python:3.7.7
+      - image: cibuilds/github:0.13 
 
     steps:
       - checkout
       - attach_workspace:
           at: .
-      - run:
-          name: Set PYODIDE_BASE_URL
-          command: PYODIDE_BASE_URL="https://cdn.jsdelivr.net/pyodide/dev/full/" make update_base_url
       - run:
           name: Install requirements
           command: |
-            sudo apt-get update
-            sudo apt-get install -y groff
+            apk add --no-cache --update python3 make
             python3 -m pip install awscli
+      - run:
+          name: Deploy Draft Github Release
+          command: |
+            cp -r build /tmp/pyodide
+            cd /tmp/
+            export CIRCLE_TAG="dev"
+            tar cjf pyodide-build-${CIRCLE_TAG}.tar.bz2  pyodide/
+            ghr -draft -t "${GITHUB_TOKEN}" -u "${CIRCLE_PROJECT_USERNAME}" -r "${CIRCLE_PROJECT_REPONAME}" -c "${CIRCLE_SHA1}" -delete  "${CIRCLE_TAG}" ./pyodide-build-${CIRCLE_TAG}.tar.bz2
```

Comment:
My earlier comment about this wasn't saved apparently.

So I think this wouldn't work because release (including draft) needs to be associated with a tag and in this case "${CIRCLE_TAG}" would be empty. Maybe we could run `ghr --help` or `ghr --version` (not sure which one would be correct according to their doc).

Reply:
Well I did attempt to account for that: see line 272 above:
`export CIRCLE_TAG="dev"`

Reply:
Yeah, but the tag wouldn't exist in git history.

Reply:
When trying that in https://github.com/pyodide/pyodide/releases/new

Says "Excellent! This tag will be created from the target when you publish this release. "

I don't think we want to create tags for each commit either. So if we release a bit frequently we will be fine, this is probably not worth it.

Reply:
So delete these lines?

Reply:
yes, let's do that.